spec/models/item_spec.rb

```diff
@@ -1,11 +1,30 @@
 require 'rails_helper'
 
-RSpec.describe Item, type: :model do
-  it 'scopes by name' do
-    item1 = create(:item, name: "Softie Jam")
-    item2 = create(:item, name: "Softie Marmelade")
+describe Item, type: :model do
+  describe 'name scoping' do
+    it 'should scope by name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.